modules/data.atmosphere/R/met.process.R

```diff
@@ -168,20 +172,28 @@ met.process <- function(site, input_met, start_date, end_date, model,
   if (stage$download.raw) {
     raw.data.site.id <- ifelse(is.null(register$siteid), new.site$id, register$siteid)
     str_ns_download <- ifelse(is.null(register$siteid), str_ns, register$siteid)
+
+    raw.id <- .download.raw.met.module(
+      dir = dir,
+      met = met,
+      register = register,
+      machine = machine,
+      start_date = start_date,
+      end_date = end_date,
+      str_ns = str_ns_download,
+      con = con,
+      input_met = input_met,
+      site.id = raw.data.site.id,
+      lat.in = new.site$lat,
+      lon.in = new.site$lon,
+      host = host,
+      overwrite = overwrite$download,
+      site = site,
+      username = username,
+      dbparms=dbparms
+    )
     
-    raw.id <- .download.raw.met.module(dir = dir,
-                                       met = met, 
-                                       register = register, 
-                                       machine = machine, 
-                                       start_date = start_date, end_date = end_date,
-                                       str_ns =str_ns_download, con = con, 
-                                       input_met = input_met, 
-                                       site.id = raw.data.site.id, 
-                                       lat.in = new.site$lat, lon.in = new.site$lon, 
-                                       host = host, 
-                                       overwrite = overwrite$download,
-                                       site = site, username = username)
-    if (met %in% c("CRUNCEP", "GFDL","NOAA_GEFS_downscale")) {
+    if (met %in% c("CRUNCEP", "GFDL","NOAA_GEFS_downscale","ERA5")) {
```

Comment:
Not your problem to solve, but more generally we should be creating a flag for this case in the registry file, rather than adding more and more special cases to the function itself.